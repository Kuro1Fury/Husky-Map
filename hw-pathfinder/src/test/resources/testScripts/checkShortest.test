# Test that the founded path is the shortest one

CreateGraph A

AddNode A n1
AddNode A n2
AddNode A n3

AddEdge A n1 n2 1.0
AddEdge A n2 n3 1.0
AddEdge A n1 n3 1.5

FindPath A n1 n3