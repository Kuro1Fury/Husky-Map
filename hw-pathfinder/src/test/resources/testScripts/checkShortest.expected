# Test that the founded path is the shortest one

created graph A

added node n1 to A
added node n2 to A
added node n3 to A

added edge 1.000 from n1 to n2 in A
added edge 1.000 from n2 to n3 in A
added edge 1.500 from n1 to n3 in A

path from n1 to n3:
n1 to n3 with weight 1.500
total cost: 1.500